ui/components-react/src/components-react/newEditors/values/Values.ts

```diff
@@ -0,0 +1,126 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Type definition for numeric value that can be handled by editor.
+ * @beta
+ */
+export interface NumericValue {
+  rawValue: number | undefined;
+  displayValue: string;
+  roundingError?: number;
+}
+
+/**
+ * Type definition for instance key value that can be handled by editor.
+ * @beta
+ */
+export interface InstanceKeyValue {
+  key: { id: string; className: string };
```

Comment:
I think using `Id64String` over `string` makes it more clear what we expect there.

```suggestion
  key: { id: Id64String; className: string };
```

Reply:
Updated type.